### ISSUE TITLE
overflow-x and overflow-y are wrongly flagged

### ISSUE THREAD

**Issue by gian1200** [ismay/stylelint-no-unsupported-browser-features#289]
After upgrading the dependency to 7.0,0, I'm facing the following error:

`Unexpected browser feature "css-overflow" is only partially supported by Safari 15.6, Safari on iOS 15.6, UC Browser for Android 15.5, QQ Browser 13.1, KaiOS Browser 2.5`

![image](https://github.com/ismay/stylelint-no-unsupported-browser-features/assets/405347/d2bc3783-7cf0-4a0d-9378-f0df9155e774)

Can I use says:
> Effectively all browsers support the CSS 2.1 definition for single-value overflow as well as `overflow-x` & `overflow-y` and values `visible`, `hidden`, `scroll` & `auto`
https://caniuse.com/?search=css-overflow

Will try to make a reproducible project later

`./.browserslistrc`:
```browserslist
defaults
not op_mini all
```

`./stylelint.config.cjs`:
```js
/**
 * @type {import("stylelint").Config}
 */
module.exports = {
	"extends": "stylelint-config-standard",
	"plugins": [
		"stylelint-no-unsupported-browser-features",
		"stylelint-stylistic"
	],
	"rules": {
		"stylistic/indentation": ["tab", { "severity": "warning" }],
		"plugin/no-unsupported-browser-features": [true, { "severity": "error" }]
	}
};
```

**Comment by ismay** [comment]
Have you checked whether the issue also occurs if you use doiuse directly?

**Comment by gian1200** [comment]
I though that checking "Can I use" page was the only thing required.

Do you mean this?
```shell
>npx doiuse@latest "src\app\index.css"
src\app\index.css:23:1: CSS Logical Properties only partially supported by: QQ Browser (13.1) (css-logical-props)
src\app\index.css:59:1: Media Queries: Range Syntax not supported by: Safari (15.6,16.3), Safari on iOS (15.6,16.0,16.1,16.2,16.3), UC Browser for Android (15.5), QQ Browser (13.1) (css-media-range-syntax)
src\app\index.css:70:1: CSS overflow property only partially supported by: Safari (15.6), Safari on iOS (15.6), UC Browser for Android (15.5), QQ Browser (13.1) (css-overflow)
src\app\index.css:71:1: CSS overflow property only partially supported by: Safari (15.6), Safari on iOS (15.6), UC Browser for Android (15.5), QQ Browser (13.1) (css-overflow)
```

Interesting, the output is the same 4 new errors I got with `stylelint-no-unsupported-browser-features@7.0.0`. 

So, should I move my report to [DoIUse](https://github.com/anandthakker/doiuse), or to which project?

Thanks

**Comment by ismay** [comment]
@gian1200 Yes if the error is identical to what you're getting from this plugin then it's coming from doiuse. This plugin leverages doiuse for checking css support, so if you feel that there is a bug in doiuse's detection then reporting it there would be the best thing to do. Once it's fixed there (assuming it's a bug), then it'll automatically be fixed for this plugin as well.

This plugin is merely the glue between doiuse and stylelint. So if anything is wrong with that integration, it's most likely this plugin. Anything that shows up as well when using doiuse directly is most likely occuring in doiuse. I'll close this for now, but feel free to let me know if it's an issue with this plugin.

**Comment by gian1200** [comment]
For future reference, the issue on [DoIUse](https://github.com/anandthakker/doiuse) is https://github.com/anandthakker/doiuse/issues/166
